packages/perseus-editor/src/components/drag-target.tsx

```diff
@@ -22,53 +22,75 @@
 // * custom styles for global drag and dragOver
 // * only respond to certain types of drags (only images for instance)!
 
-import createReactClass from "create-react-class";
-import PropTypes from "prop-types";
 import * as React from "react";
 
-const DragTarget = createReactClass({
-    propTypes: {
-        // All props not listed here are forwarded to the root element without
-        // modification.
-        onDrop: PropTypes.func.isRequired,
-        component: PropTypes.any, // component type
-        shouldDragHighlight: PropTypes.func,
-        style: PropTypes.any,
-    },
-    getDefaultProps: function () {
-        return {
-            component: "div",
-            shouldDragHighlight: () => true,
+type Props = {
+    onDrop: (e: DragEvent) => void;
+    component?: any;
+    shouldDragHighlight: (any) => boolean;
+    style?: any;
+    children?: any;
+    className?: string;
+};
+
+type DefaultProps = {
+    component: Props["component"];
+    shouldDragHighlight: Props["shouldDragHighlight"];
+};
+
+type State = {
+    dragHover: boolean;
+};
+export class DragTarget extends React.Component<Props, State> {
```

Comment:
This is already being exported as the `default` export. I think we should do one or the other. Maybe remove this and just leave the default export?  I'll leave it for you to decide. :)